src/components/app/data/utils.test.js

```diff
@@ -75,7 +75,7 @@ describe('determineLearnerHasContentAssignmentsOnly', () => {
       },
       hasCurrentEnterpriseOffers: false,
       subscriptionPlan: {
-        isActive: false,
+        isCurrent: false,
       },
       subscriptionLicense: undefined,
```

Comment:
nit: similar question here about test input data, where I'm not sure we'd actually have a `subscriptionPlan` available if `subscriptionLicense` is `undefined`.